recipe/bld.bat

```diff
@@ -1,9 +1,18 @@
 mkdir build
+if %ERRORLEVEL% neq 0 exit 1
 cd build
-cmake -GNinja ^
+
+set UNIX_LIBRARY_PREFIX=%LIBRARY_PREFIX:\=/%
```

Comment:
Why?

Reply:
there was some escape sequence error. lets recreate the error.